src/ScClient.ts

```diff
@@ -1,48 +1,62 @@
 import { invalidValue } from "./errors";
 import { ScAddr } from "./ScAddr";
 import { ScConstruction } from "./ScConstruction";
-import { ScConstructionCommand } from "./ScConstructionCommand";
-import { ScEvent, ScEventCallbackFunc } from "./scEvent";
+import { ScEvent } from "./scEvent";
```

Comment:
```suggestion
import { ScEvent } from "./ScEvent";
```
Please update for all references